package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "date-fp",
-  "version": "3.0.5",
+  "version": "3.0.6",
```

Comment:
No, I don't think so. The breaking change was made when we moved from 1.x to 2.x in #22. The fact that this function was overlooked is a bug